### DISCUSSION TITLE
Adding wordpress template to spin hub

### DISCUSSION THREAD

**Discussion by schnetzi** [serversideup/spin#140] [General]
I have created a wordpress spin template that can be used to setup a wordpress project easily. The project can be checked out here: https://github.com/schnetzi/spin-template-wordpress

Or by using the command to setup the project: `spin new schnetzi/spin-template-wordpress my-wordpress-app`

**Comment by jaydrogers** [comment]
I'll definitely circle back to this! I have a very minimal MVP published (https://serversideup.net/open-source/spin/hub), but I will make a nicer one now that someone actually wants to publish their own template 😃

I will keep you posted 👍
